src/functTest/groovy/com/bmuschko/gradle/docker/tasks/image/DockerBuildImageFunctionalTest.groovy

```diff
@@ -533,6 +551,26 @@ class DockerBuildImageFunctionalTest extends AbstractGroovyDslFunctionalTest {
             }
 
             buildTarget.finalizedBy tasks.removeImage
+
+            ${imageIdValidation()}
+        """
+    }
+
+    private static String imageIdValidation() {
+        """
+            tasks.withType(DockerBuildImage) { Task task ->
+                def assertTask = tasks.create("AssertImageIdFor\${task.name}"){
```

Comment:
What I meant here was "upper case the value of the expression $task.name". Right now the task name starts with a upper case which is not standard convention. So this should look like `assertImageIdFor\${task.name.capitalize()}`. Can you please change that?

Reply:
Yeah that makes more sense. Okay will tweak the commit now. 
Thanks for all the time for the reviews!